debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SnapshotProcessor.java

```diff
@@ -50,6 +51,7 @@
     private static final String NAME = "Snapshot Processor";
     private static final String CASSANDRA_NOW_UNIXTIMESTAMP = "UNIXTIMESTAMPOF(NOW())";
     private static final String EXECUTION_TIME_ALIAS = "execution_time";
+    private static final Set<DataType.Name> collectionTypes = new HashSet<>(Arrays.asList(DataType.Name.LIST, DataType.Name.SET, DataType.Name.MAP));
```

Comment:
Using `io.debezium.util.Collect` will simplify this a bit.

Reply:
Updated accordingly :)